infrastructure/modules/lambda_edge/main.tf

```diff
@@ -0,0 +1,92 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = ">=5.11"
```

Comment:
The `aws` provider version would be better locked to `5` e.g. `version ="~> 5.0"`

Reply:
Agreed but I think "~> 5.11.0" could be better else we would be rolling back our provider tf versioning